packages/web-app/src/controllers/review-questionnaire-submission.js

```diff
@@ -243,6 +264,35 @@ const getValues = (body) => {
   return values;
 };
 
+const getValidSelectedMissingOrIncorrect = (values) => {
```

Comment:
This function is very complex and long. Can it be split up and documented js.doc please?